### DISCUSSION TITLE
Sir. May i join on the team as a push user?

### DISCUSSION THREAD

**Discussion by Shakib-BD** [ROM-builders/temporary#23125] [General]
**My successful build** :

• AlphaDroid - Android 13 (Unofficial)
 (i) https://cirrus-ci.com/build/6552816060203008 v1.6 Vanilla
 (ii) https://cirrus-ci.com/build/6351744079233024 v1.7 Core_Gapps patch1
 (iii) https://cirrus-ci.com/build/4982207320686592 v1.7 Core_Gapps patch2
 (iv) https://cirrus-ci.com/build/5394283126063104 v1.7.9 Vanilla

• [Device Tree](https://github.com/Shakib-BD/android_device_xiaomi_merlinx/tree/alpha-13)
• [Local Manifest](https://github.com/Shakib-BD/local_manifest)

• GitHub Profile : [@Shakib-BD](https://github.com/Shakib-BD)
• Telegram : [@Shakib-BD](https://t.me/Shakib_BD)

**Comment by Apon77** [comment]
please follow #learn instruction from telegram group.

**Reply by Shakib-BD** [reply]
Okay. I will.
